f21/ps.html

```diff
@@ -1317,30 +1444,28 @@ <h2><u> Lecture / Lab 9</u></h2>
 
 
 
- <li > <b>Due Date: 20 November</b>
+ <li > <b>Due Date: 15 November</b>
  <b style="float:right">Reading: <a href="https://python-visualization.github.io/folium/docs-master/quickstart.html#Getting-Started">Folium Tutorial</a> & <a href="lab9.html">Lab 9</a></b><br>
 
 
-<p>	Using folium (see <a href="lab9.html">Lab 9</a>), write a program that asks the user for the name of a CSV file, name of the output file, and creates a map with markers for all the traffic collisions from the input file.
+<p>	Using folium (see <a href="lab9.html">Lab 9</a>), write a program that asks the user for the name of a CSV file, name of the output file, and creates a map with markers for all the NY attractions from the input file.
 
 <p>A sample run:
   <pre style = "background-color:lavender;width:400px">
-  Enter CSV file name:  collisionsThHunterBday.csv
+  Enter CSV file name:  attractions.csv
   Enter output file:  thMap.html
 </pre>
 <p>which would produce the html file:
 
-<p><iframe src="../images/th.html" width="600" height="400" frameborder="0" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+<p><iframe src="../images/map43.html" width="600" height="400" frameborder="0" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
 
-<p>(The demo above is for October 18, 2016 using the time the collision occurred ("TIME") to label each marker and changed the underlying map with the option:
+<p>Use the name of the attractions ("NAME") to label each marker and changed the underlying map with the option:
 <tt>tiles="Cartodb Positron"</tt> when creating the map.)
 
-<p> This assignment uses collision data collected and made publicly by
-<a href="https://nycopendata.socrata.com">New York City Open Data</a>.
-A sample file, <a href="../files/collisionsThHunterBday.csv">collisionsThHunterBday.csv</a> from 18 October 2016 was downloaded and can be used to test your program.
+<p>A sample file, <a href="../files/attractions.csv">attractions.csv</a> can be downloaded and used to test your program.
 
 
-<p>Note:  when creating datasets to test your program, you will need to filter for both date (to keep the files from being huge) and that there's a location entered.  The former is explained above; to check the latter, add the additional filter condition of "LONGITUDE is not blank".
+<p>Note:  when creating datasets to test your program, you will need to filter for if there's a location entered.  The former is explained above; to check the latter, add the additional filter condition of "LONGITUDE is not blank".
```

Comment:
Not sure what you mean here. Why are they creating datasets? Also, what do the former and latter refer to here? I Believe the latter is the location, but what is the former? Let's talk about this on slack and agree on how to modify this. Thanks!

Reply:
This was here from the prior semester. I can change it instead to basically say, "In your code, make sure that each row in the given dataset has a value for LONGITUDE and LATITUDE before attempting to create a marker on the map" or something like that.